apis/medusa/v1alpha1/medusaschedule_types.go

```diff
@@ -56,7 +62,7 @@ type MedusaBackupScheduleStatus struct {
 // +kubebuilder:printcolumn:name="Datacenter",type=string,JSONPath=".spec.backupSpec.cassandraDatacenter",description="Datacenter which the task targets"
 // +kubebuilder:printcolumn:name="ScheduledExecution",type="date",JSONPath=".status.nextSchedule",description="Next scheduled execution time"
 // +kubebuilder:printcolumn:name="LastExecution",type="date",JSONPath=".status.lastExecution",description="Previous execution time"
-// +kubebuilder:printcolumn:name="BackupType",type="string",JSONPath=".spec.backupSpec.backupType",description="Type of backup"
+// +kubebuilder:printcolumn:name="OperationType",type="string",JSONPath=".spec.operationType",description="Type of scheduled operation"
```

Comment:
question: Should we also have backupType still? Or has that information been made pointless earlier (I don't see PR modifying the need of that) ?

Reply:
I find it un-useful, we should hide as much as we can the notion of full backups IMO (if not fully deprecate them).
I can add it back if you want though.